src/Entity.cs

```diff
@@ -100,6 +100,30 @@ private Entity ParseEntity(dynamic response)
             {
                 entity = new Order();
             }
+            else if (response["entity"] == "customer")
+            {
+                entity = new Customer();
+            }
+            else if (response["entity"] == "invoice")
+            {
+                entity = new Invoice();
+            }
+            else if ((response["entity"] == "token") || (response.Property("deleted") != null))
+            {
+                entity = new Token();
+            }
+            else if (response["entity"] == "card")
+            {
+                entity = new Card();
+            }
+            else if (response["entity"] == "transfer")
+            {
+                entity = new Transfer();
+            }
+            else if (response.Property("transfer_id") != null)

```

Comment:
@amitmahbubani The other api's don't have a field called `transfer_id`, and reversal response doesn't contain any entity. 

https://docs.razorpay.com/v1/page/marketplace#transfersidreversals